docs/troubleshooting/common-problems.md

```diff
@@ -243,4 +243,30 @@ It should be empty, but if you have the bug, you got `check_update`.
 
 Remove `/var/lib/xcp-ng-xapi-plugins/updater.py.lock` and that should fix it.
 
----
\ No newline at end of file
+---
+
+## Disk drives not accessible when booting a live Linux ISO on a Windows VM
+
+### Cause
```

Comment:
As you did with the "Solution" part, I think having an empty line after a title make it easier to read in Markdown.

Reply:
Agree. I saw it after pushing the PR. Sorry for that... :)